### ISSUE TITLE
Migrate club case studies to github pages

### ISSUE THREAD

**Issue by thornet** [mozilla/teach.mozilla.org#1140]
To make them easier to link to after #858

**Comment by hannahkane** [comment]
Hey @thornet - can I assign this to you, or will you need help from someone else?

**Comment by thornet** [comment]
I can do it, but it'll be a P2.

**Comment by hannahkane** [comment]
Moving this to current heartbeat, so it doesn't get lost.

**Comment by hannahkane** [comment]
Moving to current heartbeat

**Comment by hannahkane** [comment]
Is this still desired? Moving to next heartbeat, but feel free to close if this is no longer needed.

**Comment by thornet** [comment]
cc @soapdog and @amirad to see if its still of interest and if Andre might
be able to assist with the migration.

On Thu, Sep 10, 2015 at 10:40 PM, hannahkane <notifications@github.com>
wrote:

> Is this still desired? Moving to next heartbeat, but feel free to close if
> this is no longer needed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/teach.webmaker.org/issues/1140#issuecomment-139372829>
> .
>



-- 
*Michelle Thorne*
Director, Mozilla Learning Programs
Mozilla Foundation

@thornet
michelle@mozillafoundation.org

**Comment by soapdog** [comment]
Yep, I can assist.

**Comment by amirad** [comment]
That would be great @soapdog

**Comment by hannahkane** [comment]
Wondering if this should be moved to current heartbeat?

**Comment by amirad** [comment]
@hannahkane can you give more info on this ticket? What club case studies?

**Comment by hannahkane** [comment]
I actually have no idea. :) Just trying to shepherd tickets along, but since I didn't create this, I don't have any more info. If it's no longer relevant / intuitive, maybe we should close?

**Comment by thornet** [comment]
If I recall, this was about moving the Google Doc versions of "how to host
a club in your...school, university, etc." to Github Pages. The idea was to
have those guides look like the rest of the club materials, rather than
editable Google Docs.

On Tue, Oct 13, 2015 at 7:24 PM, hannahkane <notifications@github.com>
wrote:

> I actually have no idea. :) Just trying to shepherd tickets along, but
> since I didn't create this, I don't have any more info. If it's no longer
> relevant / intuitive, maybe we should close?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/teach.mozilla.org/issues/1140#issuecomment-147784053>
> .
>



-- 
*Michelle Thorne*
Director, Mozilla Learning Programs
Mozilla Foundation

@thornet
michelle@mozillafoundation.org

**Comment by amirad** [comment]
Ahh okay. Thanks for the context @thornet - will proceed with it then as it makes sense. Let me see if it's doable this heartbeat or next @hannahkane

**Comment by hannahkane** [comment]
Closing this. Reopen if needed.
